odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -4398,11 +4398,11 @@ class Foo(models.Model):
         Foo._build_model(self.registry, self.env.cr)
         self.addCleanup(self.registry.__delitem__, Foo._name)
 
-        with self.assertLogs('odoo.fields', level='WARNING') as cm:
+        with self.assertLogs('odoo._orm', level='WARNING') as cm:
```

Comment:
Meh.  That namespace sucks.
We should either use `odoo.fields` for all field-specific stuff, or use prefix `odoo.orm` for all ORM-specific stuff (`odoo.orm.models`, `odoo.orm.fields`, etc.)

Reply:
I have aligned the loggers in all the files to `odoo.orm.*`.